components/twitter/actions/upload-media/upload-media.ts

```diff
@@ -0,0 +1,57 @@
+import app from "../../app/twitter.app";
+import { ACTION_ERROR_MESSAGE } from "../../common/errorMessage";
+import { defineAction } from "@pipedream/types";
+import constants from "../../common/constants";
+import fs from "fs";
+import { axios } from "@pipedream/platform";
+
+const DOCS_LINK = "https://developer.twitter.com/en/docs/twitter-api/v1/media/upload-media/api-reference/post-media-upload";
+
+export default defineAction({
+  key: "twitter-upload-media",
+  name: "Upload Media",
+  description: `Upload new media. [See docs here](${DOCS_LINK})`,
+  version: "0.0.1",
+  type: "action",
+  props: {
+    app,
+    filePath: {
+      type: "string",
+      label: "File Path",
+      description: "The file path to be uploaded.",
```

Comment:
```suggestion
      description: "The file path to upload.",
```

Reply:
Changed "to be uploaded" to "to upload" to follow Google's style guide on active voice.